tools/testsys/src/run.rs

```diff
@@ -195,6 +210,17 @@ impl Run {
             }
         };
 
+        debug!("Adding crds to testsys cluster");
+        for crd in crds {
+            let crd = client
+                .create_object(crd)
+                .await
+                .context("Unable to create object")?;
+            if let Some(name) = crd.name() {
+                info!("Successfully added '{}'", name)
+            };
```

Comment:
Isn't it safe to assume that at this point `name()` won't return `None` and thus you can do `unwrap`?